plugins/module_utils/redfish_utils.py

```diff
@@ -12,6 +12,12 @@
 from ansible.module_utils.six.moves.urllib.error import URLError, HTTPError
 from ansible.module_utils.six.moves.urllib.parse import urlparse
 
+#To support HPE operations
+import os
+from os.path import expanduser
+
+home = expanduser("~/Documents")
```

Comment:
Global variables should use upper-case letters. I would really avoid storing this as a global variable, though.

Reply:
I agree; this isn't portable to most environments.

Reply:
Appropriate changes have been made to support all environments and is available in the latest commit of the PR.